### PR TITLE
`debugRevenueCatOverlay`: added ability to display new `SubscriptionStoreView`

### PR DESCRIPTION
**Pull request by NachoSoto**
See #2593

Simulator Screenshot - iPhone 14 Pro Max - 2023-06-07 at 13 07 58

### CODE REVIEW

File: Sources/Support/DebugUI/DebugContentViews.swift [hunk 1]
```diff
@@ -175,10 +193,70 @@ private struct DebugOfferingView: View {
...
+                    Text("RevenueCat Demo Paywall")
+                }
+                .font(.title)
+
+                Text(self.offering.getMetadataValue(for: "title", default: "Premium Access"))
+                    .font(.title2)
+                    .foregroundStyle(.primary)
+
+                Text(self.offering.getMetadataValue(for: "subtitle",
+                                                    default: "Unlimited access to premium content."))
```
Comment (joshdholtz) [inline Sources/Support/DebugUI/DebugContentViews.swift:244]: 😍

**Review by joshdholtz — APPROVED** [review]

File: Sources/Support/DebugUI/DebugContentViews.swift [hunk 2]
```diff
@@ -159,9 +160,26 @@ private struct DebugSummaryView: View {
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 private struct DebugOfferingView: View {
 
+    @State private var showingSubscriptionSheet = false
+    @State private var showingStoreSheet = false
```
Comment (tonidero) [inline Sources/Support/DebugUI/DebugContentViews.swift:164]: I wonder if this could be an enum (since we can only show one of the 2 states or none right?)

**Review by tonidero — APPROVED** [review]

File: Sources/Support/DebugUI/DebugContentViews.swift (hunk 2 above)
Comment (NachoSoto) [reply Sources/Support/DebugUI/DebugContentViews.swift:164]: `SwiftUI` could handle having more than one sheet :)

Comment (tonidero) [reply Sources/Support/DebugUI/DebugContentViews.swift:164]: Oh interesting! That's fine then

**Pull request merged**